endpoints/telegram.py

```
@@ -15,25 +15,43 @@ def __init__(self, bot: telebot.TeleBot, users_service: UsersService, notificati
+                user_handle = s[2]
+                if platform == "codeforces":
+                    result = users_service.link_codeforces(user_handle, str(message.chat.id))
+                    if result == users_service.LinkCodeforcesResponse.SUCCESS:
+                        self.bot.send_message(message.from_user.id, "Codeforces account is linked. Now you can change your first name back.")
+                    elif result == users_service.LinkCodeforcesResponse.ERROR_USER_NOT_FOUND:
+                        self.bot.send_message(message.from_user.id, "Error. Codeforces account with this handle not found.")
+                    elif result == users_service.LinkCodeforcesResponse.ERROR_INCORRECT_FIRST_NAME:
+                        self.bot.send_message(message.from_user.id, f" ️Error. Please set your First Name in English in your Codeforces account to your ID ({message.chat.id}). This will help us identify you correctly. Please note that it may take up to 5 minutes to bot to see changes in your name. You can change your first name back later.")
+                else:
+                    self.bot.send_message(message.from_user.id, f"Unsupported platform '{platform}'.\nSupported platforms: codeforces")
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)